src/dasi/api/Dasi.cc

```diff
@@ -123,27 +125,28 @@ fdb5::Config construct_config(const char* dasi_config, const char* application_c
 
 //----------------------------------------------------------------------------------------------------------------------
 
-/// @note We use an internal implementation for two reasons:
-///       (i) It ensures that the interface is kept as clean as possible
-///      (ii) It ensures that no FDB headers are exposed through DASI
-
 class DasiImpl {
 
 public: // methods
-
-    DasiImpl(const char* dasi_config, const char* application_config) :
-        mainHelper_(),
-        fdb_(construct_config(dasi_config, application_config)) {}
+    DasiImpl(const char* dasi_config, const char* application_config):
+        mainHelper_(), fdb_(construct_config(dasi_config, application_config)) { }
 
     void archive(const Key& key, const void* data, size_t length) {
         fdb5::Key fdb_key;
-        for (const auto& kv : key) {
-            fdb_key.set(kv.first, kv.second);
-        }
-        LOG_DEBUG_LIB(LibDasi) << "Schema: " << fdb_.config().schemaPath() << std::endl;
+        for (const auto& kv : key) { fdb_key.set(kv.first, kv.second); }
         fdb_.archive(fdb_key, data, length);
     }
 
+    WipeGenerator wipe(const Query& query, const bool doit, const bool porcelain, const bool all) {
```

Comment:
I'm not convinced we want to expose the 'all' functionality (unless I am mis-remembering what that is). And possibly not the porcelain?

Reply:
I have misremembered what all is.